aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,178 @@
-6. On the **Settings** tab, you can select the configuration, target runtime version, and publish options. Click **Next**.
+.. _unlock-handlers:
 
-The **Preview** tab shows you the publish path (by default, the same directory as the ".sln" solution file).
 
-Deploy to IIS server
------------------------------------
+Install the .NET Core Windows Server Hosting Bundle
+---------------------------------------------------
 
-#. Navigate to the publish folder (*..\\..\\artifacts\\bin\\WebApp9\\Release\\Publish folder* in this sample).
-#. Copy the **approot** and **wwwroot** directories to the target IIS server. Note: MSDeploy is the recommended mechanism for deployment, but you can use Xcopy, Robocopy or another approach. For information on using `Web Deploy` see :doc:`iis-with-msdeploy`.
-#. In IIS Manager, create a new web site and set the physical path to **wwwroot**. You can click on **Browse *.80(http)** to see your deployed app in the browser.
+#. Install the `.NET Core Windows Server Hosting <https://microsoft.com/net>`__ bundle on the server. The bundle will install the .NET Core Runtime, .NET Core Library, and the ASP.NET Core Module. The module creates the reverse-proxy between IIS and the Kestrel server.
+#. Execute **iisreset** at the command line or restart the server to pickup changes to the system PATH.
 
-.. image:: pubIIS/_static/b8.png
+Deploy the Application
+----------------------
 
-IIS server configuration
------------------------------------
+#. On the target IIS server, create a folder to contain the application's assets.
+#. Within the folder you created, create a **logs** folder to hold application logs (if you enable logging). If plan you deploy your application with a **logs** folder in the payload, you may skip this step.
+#. Deploy the application to the folder you created on the target IIS server. MSDeploy (Web Deploy) is the recommended mechanism for deployment, but you may use any of several methods to move the application to the server (e.g., Xcopy, Robocopy, PowerShell). Visual Studio users may use the `default Visual Studio web publish script <https://github.com/aspnet/vsweb-publish/blob/master/samples/default-publish.ps1>`__. For information on using Web Deploy, see :doc:`iis-with-msdeploy`.
 
-1. Enable the **Web Server (IIS)** server role. In client operating systems (Windows 7 through Windows 10) select **Control Panel > Programs > Programs and Features > Turn Windows features on or off**, and then select **Internet Information Services**.
+.. warning:: 
+  .NET Core applications are hosted via a reverse-proxy between IIS and the Kestrel server. In order to create the reverse-proxy, the `web.config` file must be present at the content root path (i.e., the application base path) of the deployed application, which is the website physical path provided to IIS.
```

Comment:
replace i.e. with "that is" or something like
file must be present at the content root path (the app base path) of the 
or
file must be present at the content root path (typically the app base path) of the